### ISSUE TITLE
Management of auditd is missing

### ISSUE THREAD

**Issue by artem-sidorenko** [dev-sec/chef-os-hardening#182]
Tests for it are included in the linux-baseline, but we do not have any implementation within this cookbook (See #181)

**Comment by alexanderadam** [comment]
Just to be sure (as this topic is totally new for me):

I'm getting these errors now

```
Feb 19 12:34:09 some.host.name auditctl[8645]: The audit system is disabled
Feb 19 12:34:09 some.host.name auditd[8644]: Started dispatcher: /sbin/audispd pid: 8648
Feb 19 12:34:09 some.host.name audispd[8648]: priority_boost_parser called with: 4
Feb 19 12:34:09 some.host.name audispd[8648]: max_restarts_parser called with: 10
Feb 19 12:34:09 some.host.name audispd[8648]: No plugins found, exiting
Feb 19 12:34:09 some.host.name auditd[8644]: Unable to set initial audit startup state to 'enable', exiting
Feb 19 12:34:09 some.host.name auditd[8644]: The audit daemon is exiting.
Feb 19 12:34:09 some.host.name auditctl[8711]: The audit system is disables
```

Is this issue here the cause of it and is #191 the solution?

**Comment by artem-sidorenko** [comment]
@alexanderadam it looks a bit weird. Even before #191 we were just installing `auditd`with its default settings, which should be fine usually

**Comment by alexanderadam** [comment]
@artem-sidorenko this happened on Ubuntu xenial on a VPS. In case that makes any difference.

**Comment by chris-rock** [comment]
Should we rely on https://github.com/chef-cookbooks/auditd for that?

**Comment by JHeinzde** [comment]
Is help still wanted on this issue?

**Comment by chris-rock** [comment]
@artem-sidorenko Do you think we should stick to the official auditd cookbook and try to use that for configuration?

**Comment by artem-sidorenko** [comment]
@chris-rock usually yes - if it works, can be easily done and saves implementation/maintenance efforts. On the other side - if our own implementation ends up in just a simple template/service/file resource, when we should avoid the dependency from my view.

**Comment by b-dean** [comment]
Is this stuck waiting on someone to rebase #191?

**Comment by chris-rock** [comment]
@b-dean It would definitely make live for reviewers a lot easier.

**Comment by b-dean** [comment]
@chris-rock I'm not sure how I can update his PR, or if I should make another one, but I rebased @bablakely's `auditd` branch onto `master`

https://github.com/dev-sec/chef-os-hardening/compare/master...b-dean:auditd

**Comment by artem-sidorenko** [comment]
@b-dean that looks great! May I ask you to open a new PR?
